kroma-chain-ops/genesis/testdata/test-deploy-config-devnet-l1.json

```diff
@@ -51,9 +51,9 @@
   "colosseumDummyHash": "0xa1235b834d6f1f78f78bc4db856fbc49302cce2c519921347600693021e087f7",
   "colosseumMaxTxs": 100,
   "colosseumSegmentsLengths": "2,2,3,3",
-  "zkVerifierHashScalar": "0x1545b1bf82c58ee35648bd877da9c5010193e82b036b16bf382acf31bc2ab576",
-  "zkVerifierM56Px": "0x15ae1a8e3b993dd9aadc8f9086d1ea239d4cd5c09cfa445f337e1b60d7b3eb87",
-  "zkVerifierM56Py": "0x2c702ede24f9db8c8c9a439975facd3872a888c5f84f58b3b5f5a5623bac945a",
+  "zkVerifierHashScalar": "0x12F2215154ABBFE8C4CBB6CE26E741C074F0B2B7382F8DE01F7DEE61B6D83D05",
```

Comment:
I know these values in `test-deploy-config-full.json` don't affect any operations, but why not update them in that file as well?

Reply:
I updated in [87e0863](https://github.com/kroma-network/kroma/pull/321/commits/87e0863580b84dcebdc74e97992f5b473dd2b16d)